tests/integration/image-builders/release/create-release.sh

```
@@ -0,0 +1,51 @@
+
+rm -rf release-payload/blobs/sha256/*
+
+mkdir -p artifacts/staging
+
+cd artifacts/staging
+
+tar -czvf tmp_file.tar.gz ../release-manifests/
+
+ART_DIGEST=$(sha256sum tmp_file.tar.gz | cut -d ' ' -f 1)
+
+cp tmp_file.tar.gz ${ART_DIGEST}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)